### PR TITLE
feat(parrot-middleware): add content type

### PR DESCRIPTION
**Pull request by Francois-Esquire**
Add a `contentType` key for scenarios being served with `parrot-middleware`.

### CODE REVIEW

**Review by infoxicator — COMMENTED** [review]
Could you add an example of why the type is needed in the body? can this be obtained from the headers?

File: packages/parrot-middleware/README.md [hunk 1]
```diff
@@ -36,6 +36,19 @@ const scenarios = {
       },
     },
   ],
+  'has more ships with any type': [
+    {
+      request: '/ship_log',
+      response: {
+        type: 'text/plain',
```
Comment (jgolden17) [inline packages/parrot-middleware/README.md:43]: Should we go with `contentType` instead?

**Review by jgolden17 — COMMENTED** [review]
Rather than relaying on `type`, should we support response headers?

```
response: {
  body:  ....,
  headers: { .... },
  status: 200,
}
```

Then we could use `res.set()` on each header.

**Comment by Francois-Esquire** [comment]
> Could you add an example of why the type is needed in the body? can this be obtained from the headers?

@infoxicator I don't think the key `type` was meant to go in the body, it extends the API to add that key so we can set the `Content-Type` header.

File: packages/parrot-middleware/README.md (hunk 1 above)
Comment (Francois-Esquire) [reply packages/parrot-middleware/README.md]: I think that's clearer language, I've updated (a379dec)

**Comment by Francois-Esquire** [comment]
> Rather than relaying on `type`, should we support response headers?
> 
> ```
> response: {
>   body:  ....,
>   headers: { .... },
>   status: 200,
> }
> ```
> 
> Then we could use `res.set()` on each header.

@jgolden17 
That's a good idea, but we can tackle that feature at a different time.

**Review by nellyk — APPROVED** [review]

**Review by infoxicator — APPROVED** [review]

**Review by jgolden17 — APPROVED** [review]

**Pull request merged**